lib_nbgl/src/nbgl_use_case.c

```diff
@@ -361,15 +382,23 @@ static void displayReviewPage(uint8_t page, bool forceFullRefresh)
     }
 
     if (content.type == INFO_LONG_PRESS) {  // last page
-        navInfo.navWithTap.nextPageText      = NULL;
+#ifdef TARGET_STAX
+        navInfo.navWithTap.nextPageText = NULL;
+#else   // TARGET_STAX
+        navInfo.nbPages                           = navInfo.activePage + 1;
```

Comment:
I don't get this one. Why is it needed?

Reply:
Yes but why changing the `nbPages` value here? And probably this as no effect as `nbPages` should already be set to this value?

Reply:
Actually it's for the case of forward only review without known length... if we don't do that we cannot remove the '>' in the navigation bar at the last page...

Reply:
Oh! Can you add this as a comment then?

Reply:
Agree